evm/logic/system.py

```diff
@@ -125,7 +125,7 @@ def __call__(self, computation):
         call_data = computation.memory_read(start_position, size)
 
         create_msg_gas = self.max_child_gas_modifier(
-            computation.gas_meter.gas_remaining
+            computation._gas_meter.gas_remaining
```

Comment:
Shouldn't this be using `computation.get_gas_remaining()` since the intent of this PR is to make the `Computation` object the sole API that is used by all of the opcodes (and to prevent them from reaching into these internal APIs and properties.

Reply:
Yes, you are right it should use the getter, but I thought it was left intentionally, I have just changed from gas_meter to_gas_meter.

Reply:
Yes, the `computation._gas_meter` should not be accessed anywhere other than from within the `Computation` class and tests.

Reply:
Thank you, I have committed the fix